data-prepper-plugin-framework/src/main/java/org/opensearch/dataprepper/plugin/ExtensionLoader.java

```diff
@@ -59,10 +63,14 @@ public List<? extends ExtensionPlugin> loadExtensions() {
                     }
                 })
                 .collect(Collectors.toList());
-        if (!pluginErrorCollector.getPluginErrors().isEmpty()) {
+        final List<PluginError> extensionPluginErrors = pluginErrorCollector.getPluginErrors()
+                .stream().filter(pluginError -> PipelinesDataFlowModel.EXTENSION_PLUGIN_TYPE
+                        .equals(pluginError.getComponentType()))
+                .collect(Collectors.toList());
+        if (!extensionPluginErrors.isEmpty()) {
```

Comment:
Can we have one of the unit tests cover this filtering? to verify that if there are pluginErrors with type other than "extension", they will be filtered out.